terraform/modules/hub/files/tasks/hub-config.json

```diff
@@ -0,0 +1,35 @@
+[
+  {
+    "name": "config",
+    "image": "${image_and_tag}",
+    "cpu": 0,
+    "memory": 3500,
+    "essential": true,
+    "portMappings": [
+      {
+        "containerPort": 8080,
+        "hostPort": 8080
+      }
+    ],
+    "entryPoint": [
+      "/config/bin/config",
+      "server",
+      "/tmp/config.yml"
+    ],
+    "environment": [{
+      "Name": "CONFIG_DATA_PATH",
+      "Value": "/tmp/federation/configuration/config-service-data/${deployment}"
+    }, {
+      "Name": "RP_TRUSTSTORE_PASSWORD",
+      "Value": "puppet"
```

Comment:
if we are going to opensource in a future should we store any potentially sensitive data in the plain?